client/Projects/CustomNetwork/CustomNetwork/URLRequestBuilder.swift

```diff
@@ -49,6 +49,18 @@ public struct URLRequestBuilder {
 		return self
 	}
 	
+	@discardableResult
+	mutating public func addQuery(query: [String: String]) -> URLRequestBuilder {
```

Comment:
인자 네이밍이 쿼리보다 파라미터는 어떤가요~! 쿼리파라미터를 추가하는 것이니까요! ㅎㅎ
```suggestion
	mutating public func addQuery(parameters: [String: String]) -> URLRequestBuilder {
```

Reply:
반영했습니다~